src/layouts/ReferenceItemLayout.astro

```diff
@@ -283,13 +283,12 @@ const descriptionParts = description.split(
         entry.data.file && entry.data.line &&(
           <div class="my-xl">
            <div class="text-body [&_a]:text-type-magenta-dark [&_a]:!decoration-type-magenta-dark my-lg">
-            Notice any errors or typos? <a href ="https://github.com/processing/p5.js/issues">Please let us know</a>. Please feel free to edit 
+            This page is generated from the comments in
             <a
                 href={`https://github.com/processing/p5.js/blob/v${p5Version}/${entry.data.file}#L${entry.data.line}`}     
                   >
                   {entry.data.file}
-                  </a>
-                and open a pull request!
+                  </a>. Feel free to edit it and submit a pull request!
```

Comment:
I think it's alright to keep the original phrasing, "Please feel free to edit it an submit a pull request!"